features/promotion/receiving_discount/receiving_discounts_with_product_minimum_price_specified.feature

```diff
@@ -0,0 +1,73 @@
+@receiving_discount
+Feature: Receiving discounts with product minimum price specified
+    In order to pay avoid paying less than product minimum price
+    As a Visitor
+    I want to receive discount for my purchase up to product minimum price
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store classifies its products as "T-Shirts"
+        And the store has a "T-Shirt" configurable product
+        And this product has "PHP T-Shirt" variant priced at "$50.00"
+        And the "PHP T-Shirt" variant has minimum price "$45.00" in "United States" channel
+        And the store has a product "PHP Mug" priced at "$20.00"
+        And there is a promotion "Christmas sale"
+
+    @ui @api
+    Scenario: Receiving percentage discount on a single item fulfilling minimum price criteria
+        Given this promotion gives "50%" off on every product with minimum price at "$50.00"
+        When I add product "T-Shirt" to the cart
+        Then its price should be decreased by "$5.00"
+        And my cart total should be "$45.00"
+
+    @ui @api
+    Scenario: Receiving fixed discount for my cart
+        Given there is a promotion "Holiday promotion"
+        And it gives "$10.00" discount to every order
+        When I add product "T-Shirt" to the cart
+        Then my cart total should be "$45.00"
```

Comment:
you also should check discounted price of `T-shirt`

Reply:
yeah, but as its promotion for order we cant get this information on ui, so if we check for discount i had to remove @ui tag